hash_test.go

```diff
@@ -167,6 +176,8 @@ func TestHashSet(t *testing.T) {
 }
 
 func BenchmarkHashSet(b *testing.B) {
+	var testHashes = getTestHashes()
```

Comment:
I think we already have it in both functions I touched, after all the initialization.